### ISSUE TITLE
Still having CheckHealth warning despite good configuration

### ISSUE THREAD

**Issue by 10b14224cc** [mikesmithgh/kitty-scrollback.nvim#186]
I load `kitty-scrollback.nvim` using the `lazy.nvim` package manager like this:
```
-- kitty-scrollback.nvim: Open kitty buffer in nvim
return {
    "mikesmithgh/kitty-scrollback.nvim",
    enabled = true,
    lazy = true,
    cmd = {"KittyScrollbackGenerateKittens", "KittyScrollbackCheckHealth"},
    event = {"User KittyScrollbackLaunch"},
    -- version = '*', -- latest stable version, may have breaking changes if major version changed
    -- version = "^3.0.0", -- pin major version, include fixes and features that do not have breaking changes
    config = function()
        require("kitty-scrollback").setup(
            {
                -- global configuration
                {
                    checkhealth = true
                }
            }
        )
    end
}
```

However in `:CheckHealth` I still get:
```
kitty-scrollback: Kitty data ~
- WARNING No Kitty data available unable to perform a complete healthcheck
  - ADVICE:
    - Execute the command `:KittyScrollbackCheckHealth` or add the config options `checkhealth = true` to your *config* to run `checkhealth` within the context of a Kitten
    - `kitty @ kitten /home/USERNAME/.local/share/nvim/lazy/kitty-scrollback.nvim/python/kitty_scrollback_nvim.py --config ksb_builtin_checkhealth`
```

Am I doing something wrong?

**Comment by mikesmithgh** [comment]
TL;DR: you can remove `checkhealth = true` from your config and run `:KittyScrollbackCheckHealth` and it should be successful


> or add the config options `checkhealth = true` to your *config* to run `checkhealth` within the context of a Kitten

that line is a bit confusing. This means that if you have your config:
```
            {
                -- global configuration
                {
                    checkhealth = true
                }
            }
```

The next time your open kitty-scrollback.nvim it will do a healthcheck instead of open the scrollback history. So, if you press `kitty_mod+h` it will do the run check health.

I am probably going to just remove that line since it doesn't really make much sense for a normal flow.

Try running `:KittyScrollbackCheckHealth`. That should open kitty-scrollback.nvim and run check health.

If you just run `checkhealth kitty-scrollback` in a normal Neovim session, it will have the warning you are seeing because it needs extra data from Kitty for the full healthcheck. `:KittyScrollbackCheckHealth` opens kitty-scrollback.nvim so it has all the data, then performs the `checkhealth` command.

**Comment by 10b14224cc** [comment]
ok thanks

**Comment by 10b14224cc** [comment]
> The next time your open kitty-scrollback.nvim it will do a healthcheck instead of open the scrollback history.

Yes, I had understood that  it will do a healthcheck _and then_ of open the scrollback history, not _insted of_.
